tests/Microsoft.Identity.Test.Common/Core/Helpers/CoreTestConstants.cs

```diff
@@ -48,6 +48,9 @@ public static class CoreTestConstants
         public const string B2CAuthority = "https://login.microsoftonline.in/tfp/tenant/policy/";
         public const string B2CLoginAuthority = "https://sometenantid.b2clogin.com/tfp/sometenantid/policy/";
         public const string B2CRandomHost = "https://sometenantid.randomhost.com/tfp/sometenantid/policy/";
+        public const string B2CLoginAuthorityUsGov = "https://sometenantid.b2clogin.us/tfp/sometenantid/policy/";
```

Comment:
we can probably clean these up at some point. move the tenantid, tfp, and policy to constants and then form the authority.